src/components/Disputes/panels/RevealPanel.js

```diff
@@ -62,14 +86,17 @@ const RevealPanel = React.memo(function RevealPanel({
           onChange={handlePasswordChange}
         />
       </Field>
-      <Button
-        type="submit"
-        wide
-        mode="strong"
-        css={`
-          margin-bottom: ${1.5 * GU}px;
```

Comment:
Should we keep this bottom margin?

Reply:
Ah I removed it because didn't see the need for it since there's no other component rendered below `Button`